sphinx/tutorial/cylc/furthertopics/family-triggers.rst

```diff
@@ -0,0 +1,184 @@
+Family triggers
+===============
+
+This tutorial walks you through using family :term:`triggers <task trigger>`.
+
+Family triggers allow you to build dependencies based on sets of tasks
+sharing the same namespace, referred to as
+:ref:`families <tutorial-cylc-families>`.
+
+
+Purpose
+-------
+
+Family triggers are used to specify a condition to be met by a set of tasks
+in a particular :term:`family` that will :term:`trigger <task trigger>`
+the next task(s) in the :term:`graph`.
+
+Like regular task triggers, family triggers can depend on the success or
+failure of tasks. However, because a family can contain multiple tasks, you
+need to specify whether you are concerned with all or any of the tasks in
+that family reaching the desired state.
+
+
+Example
+-------
+
+Create a new suite called tutorial-family-triggers::
+
+   mkdir ~/cylc-run/tutorial-family-triggers
+   cd ~/cylc-run/tutorial-family-triggers
+
+Paste the following configuration into the ``suite.rc`` file:
+
+.. code-block:: cylc
+
+   [cylc]
+       UTC mode = True # Ignore DST
+   [scheduling]
+       [[dependencies]]
+           graph = visit_mine => MINERS
+   [runtime]
+       [[visit_mine]]
+           script = sleep 5; echo 'off to work we go'
+
+       [[MINERS]]
+           script = """
+   sleep 5;
+   if (($RANDOM % 2)); then
+       echo 'Diamonds!'; true;
+   else
+       echo 'Nothing...'; false;
+   fi
+   """
+       [[doc, grumpy, sleepy, happy, bashful, sneezy, dopey]]
+           inherit = MINERS
+
+You have now created a suite that:
+
+* Has a ``visit_mine`` task that sleeps for 5 seconds then outputs a
+  message.
+* Contains a ``MINERS`` family with a command in it that randomly succeeds
+  or fails.
+* Has 7 tasks that inherit from the ``MINERS`` family.
+
+Open the ``cylc gui`` then run the suite by pressing the "play" button
```

Comment:
Should cylc GUI be formatted to monospace font? It isn't (I believe) a command. Though if you want it to be this way maintain consistency with the other uses of 'cylc GUI' below.

Reply:
This is perhaps the main inconsistency in the documentation. Really this is a command and so should be written:
```
:command:`cylc gui`
```
Unfortunately the sphinx_rtd_theme displays commands as bold text which is somewhat confusing.

We could look at overriding this (it's only css).

Reply:
I was actually under the impression it was *not* a command, as in the tutorials the command 'gcylc' was used to open the GUI. I thought 'cylc gui' was simply being used to *refer to* the GUI. It was not made clear in the main body of the cylc tutorial, as far as I am aware, that 'cylc gui' is in fact a key/common command (it is also not explicitly listed in the command reference though I appreciate the emphasis is on the functionality of rose and not cylc independently). It would definitely be worth stating in the main body of the tutorial, before such use in the 'further topics', that 'gcylc' and 'cylc gui' can be used interchangeably for monitoring of a *single* suite (correct me if I am wrong - I am cross-referencing with the official cylc user guide here - page 29 in the LaTeX document).

Reply:
> 'gcylc' and 'cylc gui' can be used interchangeably

I've used `cylc gui` throughout the tutorial but blindly translated `gcylc` though the advanced tutorials. Will replace `gcylc` instances.

Reply:
I think (assume) you meant vice versa, as 'gcylc' was the first of these commands introduced and then singly used throughout the core tutorial, with 'cylc gui' in the further topics. Good idea to be completely consistent OR state early on that the commands are synonymous. Not quite sure why there is a need for two (usually) equivalent commands in the first place.